holder.yml

```diff
@@ -244,31 +244,29 @@ paths:
           description: Internal Error
         "501":
           description: Not Implemented
-
-  /presentations/prove:
```

Comment:
I think the group settled on `/presentations/create` ... we can check in on this during the call today.